jira_agile_metrics/querymanager.py

```diff
@@ -14,7 +14,7 @@ class IssueSnapshot(object):
     def __init__(self, change, key, date, from_string, to_string):
         self.change = change
         self.key = key
-        self.date = date.astimezone(dateutil.tz.tzutc())
+        self.date = date
         self.from_string = from_string
```

Comment:
What was the reason behind this change?

Reply:
When I run the tests on my machine, I got assertion errors in `cycletime_test.py::test_movement`. The expected calculated times where off by one hour.

I think that all dates/datetimes in `jira-agile-metrics` are timezone "naïve" (https://docs.python.org/3/library/datetime.html). I live in Switzerland, the timezone here currently is UTC+2 (Central European Summer Time CEST). I think this may have caused the error.

The highlighted line was the only part where I found a datetime that was timezone aware. Changing that fixed the test results.

I admit that there might implications that I have not yet thought about.